tests/wiseagents/test_yaml_deserializer.py

```diff
@@ -1,10 +1,13 @@
+import pytest
+
 from wiseagents import WiseAgent
 from wiseagents.llm import WiseAgentRemoteLLM
 from wiseagents.graphdb import LangChainWiseAgentGraphDB, Neo4jLangChainWiseAgentGraphDB
 import yaml
 import logging, pathlib
 
 
+@pytest.mark.skip(reason="does not pass CI/CD")
```

Comment:
The tests in this file call `llm.process_single_prompt` so I think we can mark them as `@pytest.mark.needsllm`